README.md

```diff
@@ -9,22 +9,21 @@
 This action installs the nanodu dotnet tool and udpates the NuGet packages and respective nuspec files in any .NET nanoFramework solutions living in a GitHub repository.
 
 ### Inputs
-
-|Name|Default|Description
-|--|--|--|
-`workingDirectory`|Repository workspace|Working directory. Use when targeting Solution file(s) other than the repository root.
-`branchToPr`|Name of branch to which submit the PR with the updates. Default is _main_.
-`stablePackages`|true|Use stable NuGet package versions.
-`previewPackages`|false|Use preview NuGet package versions.
-`solutionsToCheck`||List of Solution(s) to update in the `workingDirectory` directory.
-`reposToUpdate`||List of repository(es) to update.
-`exclusionList`||List of solution(s) to exclude from update. Comma separated list.
-`gitHubUser`||Git hub user for creating PR.
-`gitHubEmail`||Git hub user email for creating PR.
-`gitHubAuth`||GitHub Personal Access Token for creating PR.
-`nugetConfig`||Path to nuget.config file to use for NuGet operations.
-`repoOwner`||Repository owner on git hub. github.com/**[repoOwner]**/repositoryName
-`useTokenForClone`||Should use token for cloning repositories. Required when working with private repositories.
+|Name               |Default                            |Options                        |Description
```

Comment:
Can you please form correctly the tbles? They should all start with `|` and all finishes with `|`. For the separation line, you need at least 3 same symbols, so 3 dahses like `---` . GitHub and VS Code are tolerent but it's far to be the case of all viewers.

Reply:
Done.